heroic-elasticsearch-utils/src/main/java/com/spotify/heroic/elasticsearch/Connection.java

```diff
@@ -140,6 +141,10 @@ public SearchScrollRequestBuilder prepareSearchScroll(String scrollId) {
         return client.getClient().prepareSearchScroll(scrollId);
     }
 
+    public ClearScrollRequestBuilder clearSearchScroll(String scrollId) {
```

Comment:
Is it a problem that `scrollId` is nullable (since `lastScrollId` in the `ScrollTransformResult` class is nullable)? What does the API do when you pass it null?

Either way, adding `@Nullable` to the method signature would be helpful to set expectations.

Reply:
>What does the API do when you pass it null? 

resets the list of scrollId's to an empty array but the api will then fail the validation if the list is empty. Making this `@NotNull` instead and checking for null.